packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/constants.js

```diff
@@ -23,10 +23,10 @@ export const CLUSTER_CONFIG_TERMS = {
   PRISM_CENTRAL_TYPE: 'Type',
   PRISM_CENTRAL_VIP: 'VIP',
   PRISM_CENTRAL_IP: 'IPs',
-  PRISM_ELEMENT_VIP: 'Prism Element VIP',
-  GATEWAY_CIDR: 'Gateway Cidr',
-  REPLICATION_FACTOR: 'Replication Factor',
-  ERASURE_CODING: 'Erasure Coding',
+  PRISM_ELEMENT_VIP: 'Prism Element Virtual IP (VIP)',
```

Comment:
It was a constant before. This needs translations now. So include it in the translations file.